### PR TITLE
Corrected overview site

### LINKED ISSUE #1553: Multiple entries on overview site
**Actual Behaviour**
Multiple entries on overview site.

**Expected Behaviour**
Mutiple entries should be removed.

**Steps to reproduce it**
https://sched.eventyay.com/ check this link for the issue.

Let us know if this issue should be assigned to you or tell us who you think could help to solve this issue.
Yes

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1553

### CODE REVIEW

**Comment by agbilotia1998** [comment]
https://agbilotia1998.github.io/Webgen-overview/ - would work once site is published I've pushed the commit.

**Comment by agbilotia1998** [comment]
@geekyd done please review - https://agbilotia1998.github.io/Webgen-overview/

File: src/backend/overviewSite/index.html [hunk 1]
```diff
@@ -1,239 +1,235 @@
...
+            events. Feast your eyes on our beautiful example events.</p>
+    </div>
 </div>
 
 <div class="container-fluid bg-3 text-center">
-  <div class="row margin">
-    <div class="col-md-6 col-sm-12 col-xs-12">
-      <div class="row">
+    <div class="col-md-12 col-sm-12 col-xs-12">
         <h2 class="margin"> Apps with expandable sessions page </h2>
```
Comment (geekyd) [inline src/backend/overviewSite/index.html]: Do not centre align this follow similar design pattern as eventyay.

**Comment by agbilotia1998** [comment]
@geekyd done please review!

**Comment by aayusharora** [comment]
Looks fine for now, but we need to iterate it further on that.

**Pull request merged**